### PR TITLE
New E2 interface - RIC Control

### PR DESCRIPTION
**Pull request by shadansari**

### CODE REVIEW

File: pkg/southbound/e2.go [hunk 1]
```diff
@@ -39,10 +40,11 @@ type ControlUpdateHandler = func(sb.ControlUpdate)
...
 	EndPoint sb.Endpoint
 	Ecgi     sb.ECGI
-	client   sb.InterfaceServiceClient
+	client   e2ap.E2APClient
 
-	controlResponses chan sb.ControlResponse
-	controlUpdates   chan sb.ControlUpdate
+	ricControlRequestChan chan e2ap.RicControlRequest
+	controlResponses      chan sb.ControlResponse
+	controlUpdates        chan sb.ControlUpdate
```
Comment (SeanCondon) [inline pkg/southbound/e2.go]: can `controlResponses` and `controlUpdates` be dumped at this stage?

File: pkg/southbound/e2.go [hunk 2]
```diff
@@ -144,6 +147,35 @@ func (s *Session) manageConnection(connection *grpc.ClientConn) {
 	log.Info("Disconnected from simulator %s", s.EndPoint)
 }
 
+func (s *Session) ricControl(errors chan error) {
+	log.Info("foo 1 ricControl %s", s.EndPoint)
```
Comment (SeanCondon) [inline pkg/southbound/e2.go]: not finished yet? foo?

File: pkg/southbound/e2.go (hunk 1 above)
Comment (shadansari) [reply pkg/southbound/e2.go]: These would morph into the subscription channel which I will be working on next.

File: pkg/southbound/e2.go (hunk 2 above)
Comment (shadansari) [reply pkg/southbound/e2.go]: Oops, will fix this.

**Pull request merged**